index.html

```diff
@@ -4,9 +4,307 @@
+                    <button class="hover:underline text-red-900 font-bold text-sm">Shop HoloLens 2</button>
+                </div>
+                <div class="space-y-5">
+                    <img src="./img/business4.webp" alt="">
+                    <h3 class="text-xl font-bold">Get Microsoft Teams for free</h3>
+                    <p class="text-sm">Online meetings, chat, and up to 10 GB of shared cloud<br> storage—all in one place.</p>
+                    <button class="hover:underline text-red-900 font-bold text-sm">Sign up for free</button>
+                </div>
+            </div>
+            
+        </section>
+
+        <section class="bg-gray-200 text-center p-5 space-y-2 mb-5">
+            <h3 class="text-4xl font-bold">Microsoft's response to COVID-19</h3>
+            <p class="text-sm">Read how we’re responding to the COVID-19 outbreak, and get resources to help.</p>
+            <button class="text-sm hover:underline font-bold">Learn more</button>
+        </section>
+
+        <section class="mt-4">
+
+            <div class="slideshow-container">
+
+                <div class="mySlides2 fade">
+                    <img src="./img/cover5.webp" class="w-full relative">
+                    <div class="absolute top-20 right-0 mr-28 text-white space-y-4">
+                        <h3 class="text-4xl font-bold">Billie Jean King changes the<br> game</h3>
+                        <p class="text-sm">Microsoft partners with the tennis legend to deliver new match insights during<br> the Billie Jean King Cup by BNP Paribas Final, the Women's World Cup of<br> Tennis</p>
+                        <button class="text-black p-2 hover:underline bg-white font-bold text-sm">Learn more</button>
+                    </div>
+                    
+                </div>
+
+                <div class="mySlides2 fade">
+                    <img src="./img/cover4.png" class="w-full relative">
+                    <div class="absolute top-56 ml-28 text-white space-y-4">
+                        <h3 class="text-4xl font-bold">Go deeper into the game</h3>
+                        <p class="text-sm">See how Microsoft is helping LaLiga deliver new metrics that enable fans and<br> teams to understand the game in new and innovative ways</p>
+                        <button class="text-black p-2 hover:underline bg-white font-bold text-sm">Learn more</button>
+                    </div>
+                    
+                </div>
+                
+                <div style="text-align:center">
+                    <span class="dot"></span> 
+                    <span class="dot"></span> 
+                </div>
+
+            </div> 
+            
+        </section>
+        
+        <section class="flex justify-around w-1/5 my-5">
+            <p class="text-gray-500">Follow Microsoft</p>
+            <img class="h-8 w-8" src="./img/fb.jpg" alt="">
```

Comment:
@ktscates the Icon image is king stretched for now so I would suggest adding tailwind class called `object-cover` it just arranges well the image